http-api/src/lib.rs

```diff
@@ -1,17 +1,19 @@
 #![allow(clippy::if_same_then_else)]
+mod commit;
 mod error;
 mod project;
 
+use serde_json::json;
+use shared::keys;
```

Comment:
Can you move these two the external dependency grouping of imports?

We usually do

```
<std dependencies>

<external depdencies>

<internal dependencies>
```

Reply:
hmm yeah that was my idea too initially, but `cargo fmt --all --check` wants the `serde_json` and the `shared` crate on top..